opentreemap/treemap/templates/treemap/plot_detail.html

```diff
@@ -57,10 +58,21 @@
         {% endwith %}
     {% endfor %}
   {% endif %}
+  {% endif %}
 </div>
 
+<h3>{% trans "Planting Site Information" %}</h3>
+{% if containing_polygonalmapfeature %}
+<div class="well" id="containing-polygonalmapfeature">
+  {% with pmf_id=containing_polygonalmapfeature.pk|unlocalize %}
+  {% url 'map_feature_detail' instance_url_name=request.instance.url_name feature_id=pmf_id as polygonal_url %}
+  {% blocktrans with display_name=containing_polygonalmapfeature.display_name %}
+  This tree is part of <a href="{{ polygonal_url }}">{{ display_name }} #{{ pmf_id }}.</a> For stewardship actions, please go to the bioswale detail page.
```

Comment:
"please go to the bioswale detail page" should be something like "please go to the {{ display_name }} detail page" so the same text will work for another polygonal type. 

It is redundant, but it may be nice if the "go to..." text is also a link to to the polygonal map feature.